src/sidebar/components/sidebar-view.js

```diff
@@ -151,18 +151,14 @@ function SidebarContent({
   );
 }
 
-SidebarContent.propTypes = {
+SidebarView.propTypes = {
```

Comment:
Given that types are provided via TS-JSDocs, isn't this redundant?

Reply:
You’re right that this is rather redundant! There is some history and context to the current structure you see here. Before we introduced type checking in JSDoc comments, we relied on `propTypes` for various type-checking and type-checking-like functionality in components. `propTypes` also provides _run-time_ validation and warnings, which can be handy when developing. 

But we’ve already identified that doing both is a little much. And Rob already removed `propTypes` from `lms`:  https://github.com/hypothesis/lms/pull/1952

As I understand it, the plan is to ultimately remove `propTypes` from the `client` codebase as well. We have a few things to do before we can do that. For example, the `Button` component relies on `propTypes`’  custom validator function feature: https://github.com/hypothesis/client/blob/dea9b141459932c5065f4c96d9d626622d2d249f/src/sidebar/components/button.js#L112